modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -5,41 +5,53 @@
 [id="op-constructing-pipelines-using-pipeline-builder_{context}"]
 = Constructing Pipelines using the Pipeline Builder
 
-In the *Developer* perspective of the console, you can use the *Add* -> *Pipeline* -> *Pipeline Builder* option to:
+In the *Developer* perspective of the console, you can use the *+Add* -> *Pipeline* -> *Pipeline Builder* option to:
 
-* Construct a Pipeline flow using existing Tasks and ClusterTasks. When you install the OpenShift Pipelines Operator, it adds reusable Pipeline ClusterTasks to your cluster.
-* Specify the type of resources required for the Pipeline Run, and if required, add additional parameters to the Pipeline.
-* Reference these Pipeline resources in each of the Tasks in the Pipeline as input and output resources.
-* The parameters for a Task are prepopulated based on the specifications of the Task. If required, reference any additional parameters added to the Pipeline in the Task.
+* Configure pipelines using either the *Pipeline builder* or the *YAML view*.
+* Construct a pipeline flow using existing tasks and cluster tasks. When you install the OpenShift Pipelines Operator, it adds reusable pipeline cluster tasks to your cluster.
+* Specify the type of resources required for the pipeline run, and if required, add additional parameters to the pipeline.
+* Reference these pipeline resources in each of the tasks in the pipeline as input and output resources.
+* The parameters for a task are prepopulated based on the specifications of the task. If required, reference any additional parameters added to the pipeline in the task.
```

Comment:
This bullet isn't parallel with the other sentences; Is it possible to rewrite this so that it is similar to the other bullets? Or is this sentence actually part of the previous bullet point?

Reply:
Can we rephrase it as:

* If required, reference any additional parameters added to the pipeline in the task. The parameters for a task are prepopulated based on the specifications of the task.